kong/db/dao/targets.lua

```diff
@@ -10,7 +11,7 @@ local ipairs = ipairs
 local table = table
 local type = type
 local min = math.min
-local table_merge = utils.table_merge
+local table_merge = kong_table.table_merge
```

Comment:
Could be `local table_merge = require("kong.tools.table").table_merge`?

Reply:
updated